bug(nav): NoopAnimationsModule in @angular/material:nav test doesn't work when animation module isn't added

### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### The previous version in which this bug was not present was

_No response_

### Description

`ng add @angular/material` as of 19.2 no longer adds the angular animation module.
`ng generate @angular/material:nav`'s spec still imports the `NoopAnimationsModule`.
```
      imports: [NoopAnimationsModule]
```

I think that causes
```
./node_modules/@angular/platform-browser/fesm2022/animations.mjs:11:0-50 - Error: Module not found: Error: Can't resolve '@angular/animations/browser' in '/home/runner/work/tso-tools/tso-tools/node_modules/@angular/platform-browser/fesm2022'
```

### Reproduction

```bash
ng new test --style=scss --ssr=false
cd test
ng add @angular/material --skip-confirmation --theme purple-green --typography
ng generate @angular/material:nav room
npm test -- --no-watch --no-progress --browsers=ChromeHeadless --code-coverage
```

### Expected Behavior

tests run

### Actual Behavior

```
./node_modules/@angular/platform-browser/fesm2022/animations.mjs:11:0-50 - Error: Module not found: Error: Can't resolve '@angular/animations/browser' in '/home/runner/work/tso-tools/tso-tools/node_modules/@angular/platform-browser/fesm2022'
```

### Environment

- Angular: 19.2.?
- CDK/Material: 19.2.1
- Browser(s): ChromeHeadless
- Operating System (e.g. Windows, macOS, Ubuntu): Ubuntu

--- a/src/cdk/schematics/ng-generate/drag-drop/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.spec.ts.template
+++ b/src/cdk/schematics/ng-generate/drag-drop/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.spec.ts.template
@@ -10,7 +10,7 @@ describe('<%= classify(name) %>Component', () => {
     TestBed.configureTestingModule({
       declarations: [<%= classify(name) %>Component],
       imports: [DragDropModule]
-    }).compileComponents();
+    });
   }));<% } %>
 
   beforeEach(() => {

--- a/src/material/schematics/ng-generate/address-form/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.spec.ts.template
+++ b/src/material/schematics/ng-generate/address-form/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.spec.ts.template
@@ -1,5 +1,4 @@
-import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
-import { NoopAnimationsModule } from '@angular/platform-browser/animations';<% if(!standalone) { %>
+import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';<% if(!standalone) { %>
 import { ReactiveFormsModule } from '@angular/forms';
 import { MatButtonModule } from '@angular/material/button';
 import { MatCardModule } from '@angular/material/card';
@@ -11,23 +10,21 @@ import { <%= classify(name) %>Component } from './<%= dasherize(name) %>.compone
 
 describe('<%= classify(name) %>Component', () => {
   let component: <%= classify(name) %>Component;
-  let fixture: ComponentFixture<<%= classify(name) %>Component>;
+  let fixture: ComponentFixture<<%= classify(name) %>Component>;<% if(!standalone) { %>
 
   beforeEach(waitForAsync(() => {
-    TestBed.configureTestingModule({<% if(standalone) { %>
-      imports: [NoopAnimationsModule]<% } else { %>
+    TestBed.configureTestingModule({
       declarations: [<%= classify(name) %>Component],
       imports: [
-        NoopAnimationsModule,
         ReactiveFormsModule,
         MatButtonModule,
         MatCardModule,
         MatInputModule,
         MatRadioModule,
         MatSelectModule,
-      ]<% } %>
-    }).compileComponents();
-  }));
+      ]
+    });
+  }));<% } %>
 
   beforeEach(() => {
     fixture = TestBed.createComponent(<%= classify(name) %>Component);

--- a/src/material/schematics/ng-generate/dashboard/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.spec.ts.template
+++ b/src/material/schematics/ng-generate/dashboard/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.spec.ts.template
@@ -1,5 +1,4 @@
-import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
-import { NoopAnimationsModule } from '@angular/platform-browser/animations';<% if(!standalone) { %>
+import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';<% if(!standalone) { %>
 import { MatButtonModule } from '@angular/material/button';
 import { MatCardModule } from '@angular/material/card';
 import { MatGridListModule } from '@angular/material/grid-list';
@@ -10,22 +9,20 @@ import { <%= classify(name) %>Component } from './<%= dasherize(name) %>.compone
 
 describe('<%= classify(name) %>Component', () => {
   let component: <%= classify(name) %>Component;
-  let fixture: ComponentFixture<<%= classify(name) %>Component>;
+  let fixture: ComponentFixture<<%= classify(name) %>Component>;<% if(!standalone) { %>
 
   beforeEach(waitForAsync(() => {
-    TestBed.configureTestingModule({<% if(standalone) { %>
-      imports: [NoopAnimationsModule]<% } else { %>
+    TestBed.configureTestingModule({
       declarations: [<%= classify(name) %>Component],
       imports: [
-        NoopAnimationsModule,
         MatButtonModule,
         MatCardModule,
         MatGridListModule,
         MatIconModule,
         MatMenuModule,
-      ]<% } %>
-    }).compileComponents();
-  }));
+      ]
+    });
+  }));<% } %>
 
   beforeEach(() => {
     fixture = TestBed.createComponent(<%= classify(name) %>Component);

--- a/src/material/schematics/ng-generate/navigation/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.spec.ts.template
+++ b/src/material/schematics/ng-generate/navigation/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.spec.ts.template
@@ -1,5 +1,4 @@
-import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
-import { NoopAnimationsModule } from '@angular/platform-browser/animations';<% if(!standalone) { %>
+import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';<% if(!standalone) { %>
 import { MatButtonModule } from '@angular/material/button';
 import { MatIconModule } from '@angular/material/icon';
 import { MatListModule } from '@angular/material/list';
@@ -10,22 +9,20 @@ import { <%= classify(name) %>Component } from './<%= dasherize(name) %>.compone
 
 describe('<%= classify(name) %>Component', () => {
   let component: <%= classify(name) %>Component;
-  let fixture: ComponentFixture<<%= classify(name) %>Component>;
+  let fixture: ComponentFixture<<%= classify(name) %>Component>;<% if(standalone) { %>
 
   beforeEach(waitForAsync(() => {
-    TestBed.configureTestingModule({<% if(standalone) { %>
-      imports: [NoopAnimationsModule]<% } else { %>
+    TestBed.configureTestingModule({
       declarations: [<%= classify(name) %>Component],
       imports: [
-        NoopAnimationsModule,
         MatButtonModule,
         MatIconModule,
         MatListModule,
         MatSidenavModule,
         MatToolbarModule,
-      ]<% } %>
-    }).compileComponents();
-  }));
+      ]
+    });
+  }));<% } %>
 
   beforeEach(() => {
     fixture = TestBed.createComponent(<%= classify(name) %>Component);

--- a/src/material/schematics/ng-generate/table/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.spec.ts.template
+++ b/src/material/schematics/ng-generate/table/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.spec.ts.template
@@ -1,5 +1,4 @@
-import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
-import { NoopAnimationsModule } from '@angular/platform-browser/animations';<% if(!standalone) { %>
+import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';<% if(!standalone) { %>
 import { MatPaginatorModule } from '@angular/material/paginator';
 import { MatSortModule } from '@angular/material/sort';
 import { MatTableModule } from '@angular/material/table';<% } %>
@@ -8,20 +7,18 @@ import { <%= classify(name) %>Component } from './<%= dasherize(name) %>.compone
 
 describe('<%= classify(name) %>Component', () => {
   let component: <%= classify(name) %>Component;
-  let fixture: ComponentFixture<<%= classify(name) %>Component>;
+  let fixture: ComponentFixture<<%= classify(name) %>Component>;<% if(!standalone) { %>
 
   beforeEach(waitForAsync(() => {
-    TestBed.configureTestingModule({<% if(standalone) { %>
-      imports: [NoopAnimationsModule]<% } else { %>
+    TestBed.configureTestingModule({
       declarations: [<%= classify(name) %>Component],
       imports: [
-        NoopAnimationsModule,
         MatPaginatorModule,
         MatSortModule,
         MatTableModule,
-      ]<% } %>
-    }).compileComponents();
-  }));
+      ]
+    });
+  }));<% } %>
 
   beforeEach(() => {
     fixture = TestBed.createComponent(<%= classify(name) %>Component);

--- a/src/material/schematics/ng-generate/tree/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.spec.ts.template
+++ b/src/material/schematics/ng-generate/tree/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.spec.ts.template
@@ -17,7 +17,7 @@ describe('<%= classify(name) %>Component', () => {
         MatIconModule,
         MatTreeModule,
       ]
-    }).compileComponents();
+    });
   }));<% } %>
 
   beforeEach(() => {

